core/server/master/src/main/java/alluxio/master/file/DefaultFileSystemMaster.java

```diff
@@ -2973,6 +2984,120 @@ public void resetFile(long fileId)
     }
   }
 
+  @Override
+  public void setAcl(AlluxioURI path, SetAclAction action, List<AclEntry> entries,
```

Comment:
Do you mind adding some tests for this new method?

Reply:
I added some tests.